filters/include/pcl/filters/filter_indices.h

```diff
@@ -113,9 +113,10 @@ namespace pcl
       }
 
       template <typename Executor>
-      typename std::enable_if_t<!(std::is_same<DerivedFilter, void>::value)>
+      void
```

Comment:
It looks like you we're trying to sfinae this method out but you gave up on it. why?

Reply:
I don't think is needed, I am using a static assert now in #3